content/topics/challenges-and-prize-competitions/_index.md

```diff
@@ -10,6 +10,13 @@ title: "Challenges and Prize Competitions"
 # description — keep it short and clear
 summary: ""
 
+aliases:
+  - /topics/apps-challenges/
+  - /topics/challenge-gov/
+  - /topics/challenges/
+  - /topics/challenges-and-prize-competitions/
```

Comment:
Is this alias needed since it's the same value as the slug?

Reply:
Yes, this is redundant. Removed!